pkg/endpoint/policy.go

```diff
@@ -220,11 +220,15 @@ func (e *Endpoint) applyL4PolicyLocked(oldIdentities, newIdentities *identityPkg
 func getLabelsMap() (*identityPkg.IdentityCache, error) {
 	labelsMap := identityPkg.GetIdentityCache()
 
-	reservedIDs := policy.GetConsumableCache().GetReservedIDs()
+	reservedIDs := identityPkg.GetAllReservedIdentities()
```

Comment:
Since we only use this function here, then promptly loop over the list it returns, then resolve the numeric value back to the `Identity`, could we loop directly on the `identityPkg.ReservedIdentities` directly?
We still need to `LookupIdentityByID` unfortunately (despite all already being in `identity.reservedIdentityCache`) but it would mean not adding some code, and fixing my itch to rename `idx`.

Reply:
@raybejjani coming soon ;)

Reply:
haha. well, in that case, I look forward to the bright future ;)